src/components/animated/_liquid.less

```diff
@@ -37,12 +44,12 @@
     color: #000 !important;
     .bc(@color);
 
-    &:hover::before {
-      top: -120px;
+     &.animated:before {
```

Comment:
Why did you remove `&:hover` ? it should be for both. Something like this:

```
&:hover:before, &.animated:before
```
I also think you should combine after with it as well since it's the same rule. It'll be something like this:

```
&:hover:before, &.animated:before, &:hover:after, &.animated:after
```